### PR TITLE
Add rust sample project.

### LINKED ISSUE #59: Add sample rustc project to the repo

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #59.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ before_install:
   - chmod u+rw,go= ~/.ssh/known_hosts
   # Install dependencies to test samples projects.
   - sudo apt-get install openjdk-7-jdk --quiet --yes
+  - sudo curl -sf -L https://static.rust-lang.org/rustup.sh | sh
```
Comment (arturdryomov) [inline .travis.yml:17]: Let’s describe it better. Something like `Dependencies: Rust sample`.

**Review by arturdryomov — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (artem-zinnatullin) [reply .travis.yml:17]: Foxed

File: .travis.yml [hunk 2]
```diff
@@ -10,8 +10,11 @@ before_install:
   - chmod u+rw,go= ~/.ssh/authorized_keys
   - ssh-keyscan -t rsa localhost > ~/.ssh/known_hosts
   - chmod u+rw,go= ~/.ssh/known_hosts
-  # Install dependencies to test samples projects.
+  # Install dependencies for samples projects.
```
Comment (arturdryomov) [inline .travis.yml]: `for sample projects`

File: .travis.yml [hunk 3]
```diff
@@ -10,8 +10,11 @@ before_install:
...
+  # JDK for Gradle sample.
```
Comment (arturdryomov) [inline .travis.yml]: `## Gradle`

File: .travis.yml [hunk 4]
```diff
@@ -10,8 +10,11 @@ before_install:
...
   - sudo apt-get install openjdk-7-jdk --quiet --yes
+  # Rust build tools for Rust sample.
```
Comment (arturdryomov) [inline .travis.yml]: `## Rust`

**Review by arturdryomov — CHANGES_REQUESTED** [review]

**Review by arturdryomov — APPROVED** [review]

**Pull request merged**